content/v1.8-docs/installation/kubectl.md

```diff
@@ -8,7 +8,7 @@ description: 'cert-manager installation: Using static manifests'
 ### Prerequisites
 
 - [Install `kubectl` version `>= v1.19.0`](https://kubernetes.io/docs/tasks/tools/). (otherwise, you will have issues updating the CRDs - see [v0.16 upgrade notes](../../../docs/installation/upgrading/upgrading-0.15-0.16/#issue-with-older-versions-of-kubectl))
-- Install a [supported version of Kubernetes or OpenShift](../../installation/supported-releases/).
+- Install a [supported version of Kubernetes or OpenShift](../../../docs/installation/supported-releases.md).
```

Comment:
This link doesn't seem to work
https://github.com/SgtCoDFish/cert-manager-website/blob/linkslinkslinks/content/v1.8-docs/installation/kubectl.md#prerequisites

This is the file, I think:
https://github.com/SgtCoDFish/cert-manager-website/blob/linkslinkslinks/content/docs/installation/supported-releases.md

```suggestion
- Install a [supported version of Kubernetes or OpenShift](../../docs/installation/supported-releases.md).
```